package.json

```diff
@@ -21,6 +21,12 @@
   "homepage": "https://github.com/anenchu/rc-spanishIdValidator",
   "devDependencies": {
     "grunt": "^0.4.5",
-    "grunt-contrib-connect": "^0.9.0"
+    "grunt-contrib-clean": "^0.6.0",
+    "grunt-contrib-concat": "^0.5.1",
+    "grunt-contrib-connect": "^0.9.0",
+    "grunt-contrib-copy": "^0.8.0",
+    "grunt-contrib-uglify": "^0.9.1",
+    "grunt-ngmin": "0.0.3",
```

Comment:
I noticed that Node says ngmin is deprecated when you try to install the package but I read the document from Ana and Miiguel for the workflow in tasks. So just I put the tasks into the gruntfile. I think we can use the task annotate instead of ngmin if they are agree with this.